nginx-mainline.yaml

```diff
@@ -338,9 +338,9 @@ test:
 
 update:
   enabled: true
-  github:
-    identifier: nginx/nginx
-    use-tag: true
-    # MAINLINE VERSIONS MUST USE ODD MINOR VERSIONS (e.g., 1.27.x, 1.29.x)
-    tag-filter: release-1.29
+  # MAINLINE VERSIONS MUST USE ODD MINOR VERSIONS (e.g., 1.27.x, 1.29.x)
+  ignore-regex-patterns:
+    - ^(\d+)\.(\d+)[02468]\.(\d+)$
```

Comment:
Do we want to make the first digit of the minor version optional?

It looks like what you're doing here is ensuring that we'll match versions where the tens unit isn't even (for example 1.32) which is great.

But, if they ever release (say) a 2.2 this will fail - I think I'm less concerned about the likelihood of that than as someone taking a copy of the regex for use against a different package

I think we _should_ be able to use `^(\d+)\.((\d+)?)[02468]\.(\d+)$`